packages/dbos-cloud/cli.ts

```diff
@@ -115,16 +115,17 @@ applicationCommands
 applicationCommands
   .command('deploy')
   .description('Deploy this application to the cloud and run associated database migration commands')
-  .action(async () => {
-    const exitCode = await deployAppCode(DBOSCloudHost, false);
+  .option('--verbose', 'Verbose log of deployment step')
```

Comment:
Should this be a shared flag across all commands?

also: would it make sense to have it set a global variable so you don't have to pass it around all functions all the time?

Reply:
I think it would be good to clean up the way the logger works, right now it doesn't even have an interface that is easy to name.  If verbose set the log level to debug, we could make them debug, that'd be another way.

With regard to sharing the option, that would be good if it is implemented on all the commands... setting it as a logger level of debug would play nicely with that too.  Follow-on work?